
- name: Concatenate list with dict_values
  hosts: all
  gather_facts: False
  vars:
    mydict: {'a':1, 'b':2, 'c':3}
    mylist: [4, 5, 6]
  tasks:
  - name: Concatenate list with dict_values
    debug:
      msg: "{{ mylist + mydict.values() }}"  # This line will cause the templating error
